src/sign_workflow/signer.py

```diff
@@ -27,32 +26,27 @@ def __init__(self):
         self.git_repo.execute("./bootstrap")
         self.git_repo.execute("rm config.cfg")
 
-    def sign_artifacts(self, artifacts, basepath):
+    def sign_artifacts(self, artifacts, basepath, signature_type):
```

Comment:
There's a caller that passes 1 file and another that passes a collection. Split this function in 2, one that receives a single file and another that receives a collection.